modules/ROOT/pages/about-mule-configuration.adoc

```diff
@@ -4,8 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: configuration, deploy, mule
 
-All Mule applications, domain, and policies are configured via an XML DSL. +
-This XML file specifies the resources that compose the artifact.
+All Mule applications, domain, and policies are configured via an XML DSL.
```

Comment:
This is consolidating content from a section that was removed from the mule-application-about.adoc.

Reply:
Does "All Mule" apply to all three of "applications," "domain," and "policies"? Reword to clarify.
Note also that "all applications, domain, and policies" violates number agreement ("all domain").

Reply:
Also, should we reword for active voice: "You configure"?

Reply:
Generally, we want people configuring them through the UI. We're trying to say that underlying that is an XML.